pyefd.py

```diff
@@ -239,18 +240,55 @@ def plot_efd(coeffs, locus=(0.0, 0.0), image=None, contour=None, n=300):
     yt = np.ones((n,)) * locus[1]
 
     for n in _range(coeffs.shape[0]):
-        xt += (coeffs[n, 0] * np.cos(2 * (n + 1) * np.pi * t)) + (
-            coeffs[n, 1] * np.sin(2 * (n + 1) * np.pi * t)
-        )
-        yt += (coeffs[n, 2] * np.cos(2 * (n + 1) * np.pi * t)) + (
-            coeffs[n, 3] * np.sin(2 * (n + 1) * np.pi * t)
-        )
+        xt += coeffs[n, 0] * np.cos(2 * (n + 1) * np.pi * t) + coeffs[n, 1] * np.sin(2 * (n + 1) * np.pi * t)
+        yt += coeffs[n, 2] * np.cos(2 * (n + 1) * np.pi * t) + coeffs[n, 3] * np.sin(2 * (n + 1) * np.pi * t)
+
```

Comment:
This change works against the Black (https://github.com/psf/black) formatting and is not desired.

Reply:
Okay, I will revert this.